Benchmarks/Benchmarks.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFrameworks>net472;netcoreapp3.1</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="BenchmarkDotNet" Version="0.12.1" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\src\StackExchange.Utils.Http\StackExchange.Utils.Http.csproj" />
+  </ItemGroup>
+</Project>
```

Comment:
This change looks good, but we should nest this per conventions, e.g.:
```
benchmarks/StackExchange.Utils.Http.Benchmarks/StackExchange.Utils.Http.Benchmarks.csproj
```
Could do this after merge if no time allows - just seeing this was never merged today, woops.

Reply:
@juanrodriguezcen Did you see this one? I can relocate this post-merge too, just didn't want to step on toes.

Reply:
@NickCraver oops, I missed it, I added a new commit that moves and renames the benchmark project. Feel free to move/rename it if I didn't use the convention correctly.